global-state/jar/src/com/sixsq/slipstream/globalstate.clj

```diff
@@ -0,0 +1,83 @@
+(ns com.sixsq.slipstream.globalstate
+  "SlipStream  global state project."
+  (:require
+    [clojure.tools.logging :as log]
+    [clj-time.core :as t]
+    [qbits.spandex :as spandex]
+    [clojure.core.async :as async]
+    [environ.core :as env]
+    [com.sixsq.slipstream.scheduler :as scheduler]))
+
+
+;;define the target index for the global state
+(def ^:const index-action {:index {:_index "resources-index" :_type "global-state-snapshot"}})
```

Comment:
It probably would be better to parameterize the index to avoid having to recompile if we change it.

Reply:
OK, so I will add an environment variable like `METERING_ES_INDEX` defaulting to "resources-index"